docker/shadow-hand/indigo/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM shadowrobot/build-tools:trusty-indigo-ide
+
+COPY additional_bashrc_cmds /tmp/additional_bashrc_cmds
+
+RUN apt-get update && \
+    apt-get install -y curl && \
+    
+    echo "Running one-liner" && \
+    curl -L bit.ly/dev-machine | bash -s -- -w /home/user/projects/shadow_robot/base && \
```

Comment:
this is also something which does not look efficient. One liner should be executed in the context of the user (not only to put it in the folder in user's home directory but also rosdep will be downloaded into correct user's folder. I am using the following approach https://github.com/shadow-robot/sr-build-tools/blob/master/docker/ros/indigo/base/Dockerfile#L35-L38 . There is special command called **gosu** which will allow something to be executed in the context of the other user.